src/components/one-time-donation/RegisterDialog.tsx

```diff
@@ -28,6 +28,7 @@ export default function RegisterForm() {
     confirmPassword: formik.values.confirmPassword as string,
     terms: formik.values.terms as boolean,
     gdpr: formik.values.gdpr as boolean,
+    picture: formik.values.picture as string,
```

Comment:
```suggestion
    
```
This seems to be failing the build and is something you can leave out since we will soon change out the `one-time-donation` flow in #1272, but be sure to include some kind of default on submission of the form!